templates/primary-secondary/primary-secondary.js

```diff
@@ -46,88 +477,390 @@ class TemplatePrimarySecondary extends LitElement {
 			}
+			:host([resizable]) [data-background-shading="secondary"] .d2l-template-primary-secondary-divider,
+			:host([resizable][dir="rtl"]) [data-background-shading="primary"] .d2l-template-primary-secondary-divider {
+				box-shadow: 1px 0 0 0 rgba(0, 0, 0, 0.15);
+			}
+			:host([resizable]) [data-background-shading="primary"] .d2l-template-primary-secondary-divider,
+			:host([resizable][dir="rtl"]) [data-background-shading="secondary"] .d2l-template-primary-secondary-divider {
+				box-shadow: -1px 0 0 0 rgba(0, 0, 0, 0.15);
+			}
+			:host([resizable]) [data-background-shading="secondary"] .d2l-template-primary-secondary-divider:hover,
+			:host([resizable][dir="rtl"]) [data-background-shading="primary"] .d2l-template-primary-secondary-divider:hover {
+				box-shadow: 1px 0 0 0 rgba(0, 0, 0, 0.25);
+			}
+			:host([resizable]) [data-background-shading="primary"] .d2l-template-primary-secondary-divider:hover,
+			:host([resizable][dir="rtl"]) [data-background-shading="secondary"] .d2l-template-primary-secondary-divider:hover {
+				box-shadow: -1px 0 0 0 rgba(0, 0, 0, 0.25);
+			}
+			.d2l-template-primary-secondary-divider-handle-desktop {
+				align-items: center;
+				display: flex;
+				justify-content: center;
+				left: calc((0.65rem - 2.1rem) / 2);
+				min-height: 2.1rem;
+				min-width: 2.1rem;
+				position: absolute;
+			}
+			.d2l-template-primary-secondary-divider-handle-left,
+			.d2l-template-primary-secondary-divider-handle-right {
+				color: var(--d2l-color-celestine);
+				display: none;
+				position: absolute;
+			}
+			.d2l-template-primary-secondary-divider-handle-left {
+				left: -0.8rem;
+			}
+			.d2l-template-primary-secondary-divider-handle-right {
+				right: -0.8rem;
+			}
+			.d2l-template-primary-secondary-divider-handle-line {
+				display: flex;
+				height: 0.9rem;
+				justify-content: space-between;
+				width: 0.25rem;
+			}
+			.d2l-template-primary-secondary-divider-handle-line::before,
+			.d2l-template-primary-secondary-divider-handle-line::after {
+				background-color: var(--d2l-color-galena);
+				border-radius: 0.05rem;
+				content: '';
+				display: inline-block;
+				width: 0.1rem;
+			}
+			.d2l-template-primary-secondary-divider:focus .d2l-template-primary-secondary-divider-handle {
+				box-shadow: 0 0 0 0.1rem var(--d2l-color-celestine);
+				outline: none;
+			}
+			.d2l-template-primary-secondary-divider:focus .d2l-template-primary-secondary-divider-handle-right,
+			.d2l-template-primary-secondary-divider:focus .d2l-template-primary-secondary-divider-handle-left {
+				display: block;
+			}
+			:host(:not([dir="rtl"])) [data-is-expanded] .d2l-template-primary-secondary-divider-handle-left {
+				display: none;
+			}
+			:host([dir="rtl"]) [data-is-expanded] .d2l-template-primary-secondary-divider-handle-right {
+				display: none;
+			}
+			d2l-icon {
+				display: none;
+			}
+
 			footer {
-				background-color: #ffffff;
+				background-color: white;
```

Comment:
I think we'd talked about applying the white background to the header as well? Doesn't need to be part of this change though.